Dockerfile

```diff
@@ -3,24 +3,17 @@
 #
 # Author: Mendix Digital Ecosystems, digitalecosystems@mendix.com
 # Version: 1.4
-FROM ubuntu:trusty
+FROM mendix/rootfs
 LABEL Author="Mendix Digital Ecosystems"
 LABEL maintainer="digitalecosystems@mendix.com"
 
-#Install Python & wget
-RUN apt-get -q -y update && \
-  DEBIAN_FRONTEND=noninteractive apt-get upgrade -q -y && \
-  DEBIAN_FRONTEND=noninteractive apt-get install -q -y python wget curl libgdiplus libpq5
-# RUN apk update && \
-#     apk add --no-cache python2 curl openjdk8 postgresql-client && \
-#     apk add libgdiplus --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/
-
 # Build-time variables
 ARG BUILD_PATH=project
+ARG DD_API_KEY
 
 # Checkout CF Build-pack here
 RUN mkdir -p buildpack/.local && \
-   (wget -qO- https://github.com/mendix/cf-mendix-buildpack/archive/v1.9.1.tar.gz \
+   (wget -qO- https://github.com/mendix/cf-mendix-buildpack/archive/master.tar.gz \
```

Comment:
We should rely on released versions, so please use latest release
https://github.com/mendix/cf-mendix-buildpack/archive/v1.9.2.tar.gz

Reply:
Changed.